eos-chrome-plugin-update-upstream

```diff
@@ -85,6 +141,24 @@ fp_fetch_widevine_metadata() {
     CHROME_FILENAME=${CHROME_URL_FILENAME##*/}
 }
 
+fp_should_update_flash() {
+    if fp_fetch_flash_metadata; then
+       if [ "${FLASH_VERSION}" = "${FLASH_LKGV_VERSION}" ] ; then
+           # Touch the VERSION so it is not checked again until ${DAYS_CHECK} passed
+           echo "Flash plugin is up-to-date"
+           touch ${FLASH_VERSION_FILE}
+           return 1
+       else
+           echo "A new version of the flash plugin is available: ${FLASH_LKGV_VERSION}"
+           rm -f ${FLASH_VERSION_FILE}
```

Comment:
Same comment here as Widevine -- why remove the version file when there is an update available?